index.js

```diff
@@ -20,13 +20,10 @@ server.use(express.static(path.join(__dirname, 'app/public')));
 
 // app routes
```

Comment:
I just had an issue for ages wondering why the cookies broke when I implemented this change, then I realized it's because I put the `server.use('/', app.routes);` line before the middleware block